newsroom/agenda/agenda.py

```diff
@@ -131,11 +132,11 @@ class AgendaResource(newsroom.Resource):
     }
 
     # content metadata
-    schema["name"] = events_schema["name"]
+    schema["name"] = metadata_schema["body_html"].copy()
```

Comment:
Probably need to bump the schema versions so elastic is updated https://github.com/superdesk/newsroom-core/blob/develop/newsroom/__init__.py#L21